tests/integration/targets/junos_acls/tests/netconf/_reset_config.yaml

```diff
@@ -9,6 +9,8 @@
       - set firewall family inet filter initial_acl term ten_dot from source-port bgp protocol tcp source-address 10.0.0.0/8
       - set firewall family inet filter initial_acl term eleven_dot from source-port bgp protocol tcp source-address 11.0.0.0/8
       - set firewall family inet filter second_acl term twelve_dot from source-port bgp protocol tcp source-address 12.0.0.0/8
+      - set firewall family inet filter third_acl term thirteen_dot from source-port bgp protocol tcp source-address 13.0.0.0/16
```

Comment:
Also worth pointing out, no idea if all this had been somehow "standardized" to be /8 but at least in my head the most convenient way to stick to the grammar and still support the multi-address case was to drop to a smaller netmask.

Reply:
yeah agreed !!